server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailSetCreatePerformer.scala

```diff
@@ -128,6 +128,6 @@ class EmailSetCreatePerformer @Inject()(serializer: EmailSetSerializer,
       .flatMap(appendCommand =>
         configuration.getMaximumSendSize.toScala
           .filter(limit => appendCommand.getMsgIn.size() > limit)
-          .map(limit => Left(SizeExceededException(appendCommand.getMsgIn.size(), limit)))
+          .map(limit => Left(TooLargeException(appendCommand.getMsgIn.size(), limit)))
```

Comment:
Why do we need `TooLargeException` ? Can't we just handle `SizeExceededException` only?